hybrid/hello_ddc.c

```diff
@@ -1,6 +1,6 @@
 /***
- * This program creates a capability and writes
- * it to the Default Data Capability register.
+ * This program does some basic reads/writes
```

Comment:
Let's expand this description: what this program really does is malloc two blocks and flip the DDC to point from one to the other. In other words, it shows (at a very very very basic level) compartmentalisation.

Reply:
Done in 0f82fad.